apps/cookbook/src/app/examples/page-example/tab-navigation/page-tab-nav-example.component.ts

```diff
@@ -56,12 +57,19 @@ const config = {
 })
 export class PageTabNavExampleComponent extends BasePageExampleComponent {
   selectedIndex = 0;
-  labels = ['Item 1', 'Item 2', 'Item 3', 'Item 4', 'Item 5', 'Item6 longer'];
+  labels = [
+    'Item 1',
+    'Item 2 label text gets truncated',
+    'Item 3 label contains more text and it does not get truncated',
+    'Item 4',
+    'Item 5',
+    'Item6 longer',
+  ];
```

Comment:
```suggestion
  labels = [
    'Item 1',
    'Item 2 truncated',
    'Item 3 not truncated',
    'Item 4',
    'Item 5',
    'Item 6',
  ];
```

Consider shortening this a bit to still showcase a sensible example. I do not think we should encourage very long texts if we can avoid it 😄

Reply:
Totally agree. Your change suggestion has just enough text to get truncated, so let's use that instead 😊